src/vs/nls.js

```diff
@@ -148,7 +149,8 @@ var NLSLoaderPlugin;
                     });
                 }
                 else {
-                    req([name + suffix], messagesLoaded_1);
+                    var base = (_a = pluginConfig.baseUrl) !== null && _a !== void 0 ? _a : '';
```

Comment:
@alexdima do I revert this change and wait until the vscode-loader PR goes in and gets added to vscode?

Reply:
updated loader in core already